### PR TITLE
DOC: MacOS install instructions

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

**Comment by yarikoptic** [comment]
Cool

Just FTR: I wonder why they didn't run native annex tests and just did few custom ones

**Comment by mih** [comment]
@yarikoptic who is they?

**Comment by yarikoptic** [comment]
brew ppl -- I just looked into their build recipe for git-annex: https://github.com/Homebrew/homebrew-core/blob/master/Formula/git-annex.rb#L47

File: docs/source/faq.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+.. _chap_faq:
+
+**************************
+Frequently asked questions
+**************************
+
+I have no permission to install software on a server I want to use datalad on. Can I make it work nevertheless?
+  ``pip`` supports installation into a user's home directory with ``--user``.
+  Git-annex, on the other hand, can be deployed by extracting a pre-built
+  binary from a tarball (that also includes an up-to-date Git installation).
```
Comment (yarikoptic) [inline docs/source/faq.rst]: s/binary/binaries/

File: docs/source/faq.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  `Obtain the tarball <https://downloads.kitenet.net/git-annex/linux/current/>`_,
```
Comment (yarikoptic) [inline docs/source/faq.rst]: isn't situation the same on OSX, i.e. better just to point to generic installation page?

Comment (mih) [inline docs/source/faq.rst]: There are no OSX servers ;-)

**Pull request merged**